docs/releases/3.1.txt

```diff
@@ -347,6 +347,10 @@ Models
 * :meth:`.QuerySet.bulk_create` now sets the primary key on objects when using
   MariaDB 10.5+.
 
+* The ``DatabaseOperations.sql_flush()`` method now generates more efficient
+  SQL for table containing a few rows on MySQL by using
+  ``DELETE`` statements instead of ``TRUNCATE``.
```

Comment:
```suggestion
* The ``DatabaseOperations.sql_flush()`` method now generates more efficient
  SQL for table which doesn't need to reset sequences on MySQL by using
  ``DELETE`` statements instead of ``TRUNCATE``.
```